index.js

```diff
@@ -0,0 +1,4 @@
+const cmd_loop = require('./command_loop');

```

Comment:
Good index 👍

If I were to be super picky, I'd be tempted to have the index import the commands and pass the command dictionary to `cmd_loop.begin(command_dict)`.

That'd make extending it in future slightly more obvious (your index defines the commands you want to use), and it cleans the dependencies up by making command_loop a generic command runner that can take arbitrary commands (as long as they conform to its expectations about arguments/return values).